### PR TITLE
add additional logging to redeploy vm e2e

### PR DESCRIPTION
**Pull request by bryanro92**
### Which issue this PR addresses:

more logging for redeploy vm e2e. doesn't fix anything, but at least adds some more info for when this test fails.

### What this PR does / why we need it:
adds additional logging to redeploy vm e2e to better understand state when this job fails

### Test plan for issue:
n/a

### Is there any documentation that needs to be updated for this PR?
n/a

### CODE REVIEW

**Review by petrkotas — DISMISSED** [review]
Good thinking, LGTM. The e2e is failing on a flaky test unfortunately. Lets rerun it.

**Review by bennerv — COMMENTED** [review]
Do we care to put the node name in the log line at all?  We can dig the information out but might be nice to have it available right there.

**Review by s-amann — APPROVED** [review]
+1 to add the name of the node to the log

File: test/e2e/adminapi_redeployvm.go [hunk 1]
```diff
@@ -67,19 +68,18 @@ var _ = Describe("[Admin API] VM redeploy action", func() {
 			var nodeNotReady, rebooted, nodeReady bool
 
 			for _, event := range events.Items {
-				if event.CreationTimestamp.After(nodeKillTime.Time) {
-					if !nodeNotReady &&
-						event.Reason == "NodeNotReady" &&
-						event.Regarding.Name == *vm.Name {
+				if event.CreationTimestamp.After(nodeKillTime.Time) && event.Regarding.Name == *vm.Name {
```
Comment (bryanro92) [inline test/e2e/adminapi_redeployvm.go:71]: I've moved a repeat conditioin of `event.Regarding.Name == *vm.Name` into this parent conditional

**Review by s-amann — APPROVED** [review]

**Review by m1kola — APPROVED** [review]

**Pull request merged**